prac_06/languages.py

```diff
@@ -0,0 +1,20 @@
+"""
+CP1404 Practical - Client code to use the ProgrammingLanguage class.
+Expected time = 30 minutes
+Actual time = 20 minutes
+"""
+
+from programming_language import ProgrammingLanguage
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+# print(python)
+# print(ruby)
+# print(visual_basic)
+
+programming_languages = [python, ruby, visual_basic]
+print("The dynamically typed languages are:")
+for language in programming_languages:
+    if ProgrammingLanguage.is_dynamic(language):
```

Comment:
if you want you can format the method as language.is_dynamic() without the class's name, though it is good either way

Reply:
Jack's suggestion is good - please use the normal concise syntax, not this unbound version.